examples/docs_beta_snippets/docs_beta_snippets/guides/data-assets/data-assets/asset_docorator.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+import dagster as dg
+
+
+def load_data() -> str:
+    # Load data from S3, the web, etc.
+    ...
+
+
+def transform_data(data: str) -> List[str]:
+    # Transform the data some way
+    ...
+
+
+def store_data(files: List[str]):
+    # Store the data somewhere
+    ...
+
+
+@dg.asset(
+    owners=["bighead@hooli.com", "team:roof", "team:corpdev"],
+)
```

Comment:
as noted above, this example should probably contain two assets, with one downstream of the other